homework2/reviewapp/src/components/shared/StarsRating/StarsRating.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box } from "@chakra-ui/react";
+import {StarIcon} from "@chakra-ui/icons"
+import React from "react";
+
+interface IStarsRatingProps {
+    selected: boolean;
+    onChange: (selected: boolean) => void;
+  }
+  
+  export default function StarIconComponent({ selected, onChange }: IStarsRatingProps) {
```

Comment:
Couple of things here.
We usually just export our components (we leave default export only for pages).
I would rather name this component smth. like `StarRatingInput` and handle rendering all 5 stars here with all the necessary info.
With a few tweaks you can use it inside form and review item :)
I think we even discussed it on consultation :)
If you don't remember, we can go over this together, just give me a ping :)

Reply:
I implemented the changes